examples/example_FastMRI_Unet.py

```diff
@@ -0,0 +1,206 @@
+# %%
+"""
+Simple UNet model.
+
```

Comment:
Please make sure to give the title in the right  wayso that its picked up. Please follow https://github.com/mind-inria/mri-nufft/blob/master/examples/GPU/example_learn_samples.py


https://github.com/mind-inria/mri-nufft/blob/092c795c9fc251966ce0422ee40eb4419ff4faf1/examples/GPU/example_learn_samples.py#L1-L21

Also, write your math for what is being learnt.